nautobot/extras/management/commands/remove_stale_scheduled_jobs.py

```diff
@@ -0,0 +1,22 @@
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from django.utils import timezone
+
+from nautobot.extras.choices import JobExecutionType
+from nautobot.extras.models import ScheduledJob
+
+
+class Command(BaseCommand):
+    help = "Remove stale scheduled jobs."
+
+    def add_arguments(self, parser):
+        parser.add_argument("max-age", type=int, help="Max age in days")
```

Comment:
Did we want to set a default for `max-age`? If not we should set `required=True` here.